packages/wonder-blocks-clickable/src/components/__docs__/clickable.stories.js

```diff
@@ -0,0 +1,296 @@
+        ): any),
+        docs: {
+            description: {
+                component: null,
+            },
+            source: {
+                // See https://github.com/storybookjs/storybook/issues/12596
+                excludeDecorators: true,
+            },
+        },
+    },
+};
+
+export const Default: StoryComponentType = (args) => (
+    <Clickable {...args}>
+        {({hovered, focused, pressed}) => {
+            return (
+                <View
+                    style={[
+                        styles.clickable,
+                        hovered && styles.hovered,
+                        pressed && styles.pressed,
+                    ]}
+                >
+                    <Body>This text is clickable!</Body>
+                </View>
+            );
+        }}
+    </Clickable>
+);
+
+export const Basic: StoryComponentType = () => (
+    <View style={styles.centerText}>
+        <Clickable
+            href="https://www.khanacademy.org/about/tos"
+            skipClientNav={true}
+        >
+            {({hovered, focused, pressed}) => (
+                <View
+                    style={[
+                        hovered && styles.hovered,
+                        pressed && styles.pressed,
+                    ]}
+                >
+                    <Body>This text is clickable!</Body>
+                </View>
+            )}
+        </Clickable>
+    </View>
+);
+
+Basic.parameters = {
+    docs: {
+        description: {
+            story: "You can make custom components Clickable by returning them in a function of the Clickable child. The eventState parameter the function takes allows access to states pressed, hovered and clicked, which you may use to create custom styles.\n\nClickable has a default focus ring style built-in.  If you are creating your own custom focus ring it should be disabled using by setting `hideDefaultFocusRing={true}` in the props passed to `Clickable`.",
+        },
+    },
+    chromatic: {
+        // we don't need screenshots because this story is already covered in
+        // `Default`. We add this story to the `Docs` tab to present the
+        // description above along with the example.
+        disableSnapshot: true,
+    },
+};
+
+/**
+ * Clickable usage on dark backgrounds
+ */
+export const Light: StoryComponentType = () => (
+    <View style={styles.dark}>
+        <Clickable
+            href="https://www.khanacademy.org/about/tos"
+            skipClientNav={true}
+            light={true}
+        >
+            {({hovered, focused, pressed}) => (
+                <View
+                    style={[
+                        styles.clickable,
+                        hovered && styles.hovered,
+                        pressed && styles.pressed,
+                    ]}
+                >
+                    <Body>This text is clickable!</Body>
+                </View>
+            )}
+        </Clickable>
+    </View>
+);
+
+Light.parameters = {
+    chromatic: {
```

Comment:
Would we still want the chromatic snapshot to test the visuals of the light text on a dark background?

Reply:
The thing here is that the light text is added outside of the `Clickable` instance, so the snapshot won't be able to indicate if the Clickable internally set things right, as the only thing that internally changes is the focus ring (that doesn't appear on load).